### PR TITLE
fix: upgradehandler v1.2.10

### PR DESCRIPTION
**Pull request by ajansari95**
- Update v1.2.10 upgradehandler to ensure call to close channel originates in ics keeper.

### CODE REVIEW

File: app/upgrades.go [hunk 1]
```diff
@@ -155,27 +153,11 @@ func v010209UpgradeHandler(app *Quicksilver) upgradetypes.UpgradeHandler {
 
 func v0102010UpgradeHandler(app *Quicksilver) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-		_, capability, err := app.InterchainstakingKeeper.IBCKeeper.ChannelKeeper.LookupModuleByChannel(ctx, "icacontroller-stargaze-1.delegate", "channel-50")
+		err := app.InterchainstakingKeeper.CloseICAChannel(ctx, "cacontroller-stargaze-1.delegate", "channel-50")
```
Comment (joe-bowman) [inline app/upgrades.go:156]:
```suggestion
		err := app.InterchainstakingKeeper.CloseICAChannel(ctx, "icacontroller-stargaze-1.delegate", "channel-50")
```

**Review by joe-bowman — APPROVED** [review]
lgtm - just testing

**Review by joe-bowman — APPROVED** [review]
lgtm - just testing

**Review by aljo242 — DISMISSED** [review]

**Review by joe-bowman — APPROVED** [review]
lgtm

**Review by joe-bowman — APPROVED** [review]
lgtm

**Review by aljo242 — APPROVED** [review]

**Review by aljo242 — APPROVED** [review]

File: x/interchainstaking/keeper/keeper.go [hunk 2]
```diff
@@ -515,6 +515,31 @@ func (k *Keeper) Rebalance(ctx sdk.Context, zone types.Zone, epochNumber int64)
 	return k.SubmitTx(ctx, msgs, zone.DelegationAddress, fmt.Sprintf("rebalance/%d", epochNumber), zone.MessagesPerTx)
 }
 
+func (k *Keeper) CloseICAChannel(ctx sdk.Context, portID, channelID string) error {
```
Comment (joe-bowman) [inline x/interchainstaking/keeper/keeper.go:518]: this could go now actually

Comment (ajansari95) [reply x/interchainstaking/keeper/keeper.go:518]: let me remove it

**Pull request merged**